Dockerfile

```diff
@@ -53,7 +53,8 @@ ENV PATH="/home/limitador/bin:${PATH}"
 COPY --from=limitador-build /usr/src/limitador/limitador-server/examples/limits.yaml ../
 COPY --from=limitador-build /usr/src/limitador/target/release/limitador-server ./limitador-server
 
-RUN chown limitador:limitador limitador-server
+RUN chown -R limitador:limitador /home/limitador \
+    && chmod -R 750 /home/limitador
```

Comment:
can we add a comment here pointing to the explanation about why permissions need to be set up this way, for the future maintainer ?